sfdoc/publish/models.py

```diff
@@ -305,4 +305,4 @@ def all_urls(cls):
     @property
     def urllist(self):
         """Urls as a list instead of a string"""
-        return self.urls.split("\n")
+        return [url.strip() for url in self.urls.split("\n")]
```

Comment:
You could have done self.urls.splitlines(), but I like that this also strips any stray whitespace.